Lombiq.HelpfulLibraries/Libraries/Contents/ContentExtensions.cs

```diff
@@ -138,5 +138,31 @@ public static async Task SanitizeContentItemVersionsAsync(this IContent content,
             $"DisplayText: {content.ContentItem.DisplayText}, " +
             $"ID: {content.ContentItem.ContentItemId}, " +
             $"Version ID: {content.ContentItem.ContentItemVersionId}";
+
+        /// <summary>
+        /// Returns the most relevant date of the <paramref name="content"/>'s <see cref="ContentItem"/>.
+        /// </summary>
+        /// <returns>
+        /// <para>
+        /// The values are resolved in the following order if available. If all of them are <see langword="null"/> then
+        /// <see cref="DateTime.MinValue"/> is returned.
+        /// </para>
+        /// <list type="bullet">
+        ///     <item>
+        ///         <description><see cref="ContentItem.ModifiedUtc"/></description>
+        ///     </item>
+        ///     <item>
+        ///         <description><see cref="ContentItem.PublishedUtc"/></description>
+        ///     </item>
+        ///     <item>
+        ///         <description><see cref="ContentItem.CreatedUtc"/></description>
+        ///     </item>
+        /// </list>
+        /// </returns>
+        public static DateTime GetDateTime(this IContent content) =>
+            content.ContentItem.ModifiedUtc ??
+            content.ContentItem.PublishedUtc ??
+            content.ContentItem.CreatedUtc ??
+            DateTime.MinValue;

```

Comment:
I wanted to suggest to return a UTC DateTime here, too, but [it turns out that](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.minvalue?view=net-5.0#remarks):

> The value of this constant is equivalent to 00:00:00.0000000 UTC, January 1, 0001, in the Gregorian calendar.

😀

Reply:
_In the year of our lord and savior, Universal Time Christ..._

That makes a lot more sense than UTC somehow being an acronym for "Coordinated Universal Time". 😄